### PR TITLE
ALESCo Meeting 10/02/2024 Minutes

### PR DESCRIPTION
**Pull request by codyro**
Excuse the belated publishing.

### CODE REVIEW

File: docs/alesco/meeting-minutes/2024-10-02.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+### Rebuilding EPEL for x86_64 v2
+- Cody: Introduced the topic of rebuilding EPEL for the x86_64 v2 architecture. Noted that previous votes focused on v3 and there was ambiguity regarding v2.
+- Andrew: Expressed concerns about the potential workload and maintenance challenges.
+  - Highlighted the need to adjust macro systems and spec files.
+	- Did not want to maintain changes or forks on our side.
+  - Jonathan: Clarified that the build system can reasonably supports this
+    - Andrew: Yes, not a problem
+	- Neil: Pointed out that Fedora still builds for v1 and EPEL packages are Fedora backports, so issues might be minimal.
+	- Neil:	Emphasized the importance of EPEL for the usability of RHEL-based distributions.
+	- Neil:	Suggested that releasing without EPEL support might hinder adoption.
```
Comment (Conan-Kudo) [inline docs/alesco/meeting-minutes/2024-10-02.md:39]: `Neil` -> `Neal`

File: docs/alesco/meeting-minutes/2024-10-02.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	- Alex: Agreed with Andrew’s concerns.
+  - Neil: Can do this in stages, thinks we should approve it and see demand to make further decisions
+	- Jonathan: Proposed tabling the decision to gather more data on user demand and revisit the topic after the release of AlmaLinux Kitten.
+	- Neil: Warned that releasing AlmaLinux 10 without EPEL support for x86_64 v2 could impede adoption/usage
```
Comment (Conan-Kudo) [inline docs/alesco/meeting-minutes/2024-10-02.md:43]: Ditto

File: docs/alesco/meeting-minutes/2024-10-02.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+	  - Suggested that if they decide not to build EPEL for v2, they should consider not releasing the v2 variant at all.
+	-	Decision: The team agreed to table the motion and reassess after AlmaLinux Kitten is released.
+
+### GitHub Repository for Official Changes/RFCs
+
+-	Cody: Presented a PoC for a GitHub repository to manage Requests for Comments (RFCs) and official changes [1]
+  - Inspired by ArchLinux and Rust RFC processes.
+	-	Neil: Suggested replacing the “Alternatives” section with a “Feedback” section to capture community input.
+	- Neil:	Emphasized that submitting an RFC does not obligate the ALESCo committee to implement it; contributors should be responsible for the work.
+	-	Neil: Proposed adding sections about the benefit to AlmaLinux and feedback from the community/marketing perspective
```
Comment (Conan-Kudo) [inline docs/alesco/meeting-minutes/2024-10-02.md:53]: Ditto.

File: docs/alesco/meeting-minutes/2024-10-02.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+	-	Jonathan: Agreed with the proposed changes
+	-	Jonathan: Discussed the possibility of mentoring contributors who might need assistance.
+	-	Jonathan: Suggested directing contributors to the appropriate SIGs (Special Interest Groups).
+	-	Cody: Agreed to incorporate the feedback and adjust the RFC template accordingly and share the updated template for further input
+	- Cody: Mentioned the need for an example template using a past decision.
+    - Neil: Jonathan and him can draft up the x86_64_v2 one as an example
```
Comment (Conan-Kudo) [inline docs/alesco/meeting-minutes/2024-10-02.md:59]: Ditto

**Review by Conan-Kudo — CHANGES_REQUESTED** [review]

**Review by Conan-Kudo — APPROVED** [review]

**Review by jonathanspw — APPROVED** [review]

**Review by LKHN — APPROVED** [review]

**Comment by jonathanspw** [comment]
poke @andrewlukoshko @bennyvasquez

**Review by bennyvasquez — APPROVED** [review]

**Review by andrewlukoshko — APPROVED** [review]

**Pull request merged**